### PR TITLE
[MIG] portal_sale_distributor: Migration to 18.0

### PR DESCRIPTION
**Pull request by lav-adhoc**

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/sale/pull/983).

**Comment by nicolascol** [comment]
@roboadhoc r+

**Comment by roboadhoc** [comment]
I'm sorry, @nicolascol. I'm afraid I can't do that.

File: portal_sale_distributor/models/partner.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    credit_limit = fields.Float(
+        groups='account.group_account_invoice,account.group_account_readonly,portal_sale_distributor.group_portal_backend_distributor')
```
Comment (matiasperalta1) [inline portal_sale_distributor/models/partner.py:10]: borrar ese enter

Comment (matiasperalta1) [reply portal_sale_distributor/models/partner.py:10]: e identar

**Review by matiasperalta1 — CHANGES_REQUESTED** [review]

**Comment by matiasperalta1** [comment]
@roboadhoc r+

**Pull request closed without merging**